frontend/src/components/panels/Document.vue

```diff
@@ -312,6 +328,23 @@ export default defineComponent({
 			if (rulerHorizontal) rulerHorizontal.handleResize();
 			if (rulerVertical) rulerVertical.handleResize();
 		},
+		async pasteFile(e: DragEvent) {
```

Comment:
Does this need to be `async`? I don't see any `await` prefixes in the function implementation. Excuse me if I'm missing something obvious which is probably the case.